src/accordian.js

```diff
@@ -0,0 +1,13 @@
+ export default function() {
+  var btn = $('.accordion__bar');
+  var list = $('.accordion__bar-list');
+
+  btn.on('click', function() {
+    $(this).next(list).slideToggle();
```

Comment:
thanks man a part of it is I'm just not thinking clear right now thanks for the support and all the help it does mean alot I'm pulling myself out of this crap so I can focus on this assignment thank you ryan!!!

Sent from Yahoo Mail on Android

From:"Ryan Tablada" <notifications@github.com>
Date:Sun, Nov 15, 2015 at 1:58 PM
Subject:Re: [15-jquery-ui] turning in assignment 15-jquery-ui (#1)

In src/accordian.js:

> @@ -0,0 +1,13 @@ > + export default function() { > + var btn = $('.accordion__bar'); > + var list = $('.accordion__bar-list'); > + > + btn.on('click', function() { > + $(this).next(list).slideToggle(); 

Try not to use the next chain. It may lead to unexpected results.

Instead look at using siblings.

This will add some extra steps to find things in the DOM, but will be clearer what your intent is.

—
Reply to this email directly or view it on GitHub.￼